CONFIG.md

```diff
@@ -355,6 +355,13 @@ This is an example of the settings for the validate tasks:
 
 ```
 {
+  "eslint": {
+    "srcPaths": [
```

Comment:
We've called this "dir" for PHPCS, should we try to make the property names parallel?